benchmark/test_runner/suite-tests.js

```diff
@@ -52,8 +49,6 @@ async function run({ numberOfSuites, testsPerSuite, testType, concurrency }) {
     }
   }
 
-  await finished(reporter);
```

Comment:
why, how can we make sure that the tests finished?

Reply:
I initially did understand this was actually running in the same process, and didnt undersand how this worked - so I am reverting this as well